source/blog/2018-12-07-the-ember-times-issue-76.md

```diff
@@ -45,8 +45,15 @@ Jingle bells, jingle bells, **DecEmber** all the way! 🐹🎄
 
 ---
 
-## [SECTION TITLE](#section-url)
+## [JSON:API Update 1.1 🇯](https://twitter.com/jsonapi/status/1069593631365959680)
+
+**JSON:API 1.1 RC1** is released! 🎉🎊🎈 The spec will hopefully be finalized by January 31 which is very exciting! If you use JSON:API and are able to, try out the upcoming version and give some feedback.
+
+One notable feature is the addition of **"profiles"** which allow specifying addition semantics on top of the current spec. This could allow specifying things like filtering, pagination and more. Check out [profiles here](https://jsonapi.org/extensions/).
+
+Another noteworthy change is the official recommendation to use camelCase for resource fields. This standardization will help allow composing and parsing **JSON:API** with JavaScript and other languages. Check it out [here](https://jsonapi.org/recommendations/#naming).
```

Comment:
Just a small one here, as I think it would be:

```suggestion
Another noteworthy change is the official recommendation to use camelCase for resource fields. This standardization will allow composing and parsing **JSON:API** with JavaScript and other languages. Check it out [here](https://jsonapi.org/recommendations/#naming).
```